### ISSUE TITLE
question about fm

### ISSUE THREAD

**Issue by seun-wun** [yxtay/recommender-tensorflow#1]
Does line91 in <recommender-tensorflow/src/models/deep_fm.py> means that 'mf_logit' is a predicted value of our label with the mf method? But I can not see the method of FM in this section of code? Why 
0.5 * tf.reduce_sum(sum_square - square_sum, 1, keepdims=True)  represents for the predicted value?
